code/datums/gas_mixture.dm

```diff
@@ -694,14 +672,12 @@ What are the archived variables for?
 //Does handle trace gases!
 
 /datum/gas_mixture/proc/get_breath_partial_pressure(gas_pressure)
-	var/breath_pressure = (total_moles()*R_IDEAL_GAS_EQUATION*temperature)/BREATH_VOLUME
-	return (gas_pressure/total_moles())*breath_pressure
+	return (gas_pressure*R_IDEAL_GAS_EQUATION*temperature)/BREATH_VOLUME
 
 
 //Reverse of the above
 /datum/gas_mixture/proc/get_true_breath_pressure(breath_pp)
-	var/breath_pressure = (total_moles()/R_IDEAL_GAS_EQUATION/temperature)*BREATH_VOLUME
-	return (breath_pp/breath_pressure*total_moles())
+	return (breath_pp*BREATH_VOLUME)/(R_IDEAL_GAS_EQUATION*temperature)

```

Comment:
Not only was the math here wrong (these two procs returned the same value given the same argument) but there were two calls to `total_moles()` that cancelled each other out because fractions